product_docs/docs/pgd/4/rel_notes/pgd_4.3.1_rel_notes.mdx

```diff
@@ -0,0 +1,24 @@
+---
+title: "Release notes for EDB Postgres Distributed version 4.3.1"
+navTitle: "Version 4.3.1"
+---
+
+EDB Postgres Distributed version 4.3.1 is a patch release of EDB Postgres Distributed 4, which includes bug fixes for issues identified in previous versions.
+
+!!! Note
+    This version is required for EDB Postgres Advanced Server versions 12.14.18, 13.10.14, 14.7.0 and later.
+
+| Component | Version |  Type           |                                                   Description                                                           |
+| --------- | ------- | --------------- | ------------------------------------------------------------------------------------------------------------------------|
+| BDR       | 4.3.1   | Feature         | Add pid to the log message emitted upon writer process death (BDR-2534) |
+| BDR       | 4.3.1   | Bug fix         | Fix join and replication when SDW and standby_slot_names are set (BDR-3176, RT89702, RT89536) |
+| BDR       | 4.3.1   | Bug fix         | Fix spurious warnings when processing sequence options (BDR-3295, RT90668) |
+| BDR       | 4.3.1   | Bug fix         | Fix upgrades for nodes with CRDTs (BDR-3302) |
+| BDR       | 4.3.1   | Bug fix         | Fix replication for sub group nodes (BDE-3204, RT89814)  If a node had been part of a sub-group, then parted and then rejoined to the group, it could be inconsistent with the group. The changes from some nodes of the group would be replayed from a wrong starting point, resulting in potential data-loss. |
+| BDR       | 4.3.1   | Bug fix         | Adjust lag tracking parameters for LCRs from pg_stat_replication (BDR-3040) |
+| BDR       | 4.3.1   | Bug fix         | Convert all sequences to "distributed" during create node (BDR-2890) |
+| BDR       | 4.3.1   | Bug fix         | Fix streaming transactions with standby_slot_names (BDR-3248)  This could have led to a subscriber-only node get ahead of a data-node.|
+| BDR       | 4.3.1   | Bug fix         | Free memory consumed by dlist node holding MQConn (BDR-3409, RT91830)  The memory consumed by the node is merely 32 bytes but when the consensus worker handles hundreds of requests per second, sustained for hours, the memory builds up. We saw 47% of memory consumed by consensus worker when used with HARP, which executes bdr.consensus_kv_fetch() at a rate of 600 times per second. |
+| BDR       | 4.3.1   | Bug fix         | Make Raft leader step down on loss of quorum  (isolated) |
+| BDR       | 4.3.1   | Bug fix         | Avoid restarting sync workers.(BDR-3415)  This is to prevent the node join from failing when config changes are made that signal the restart of subscription workers. |
```

Comment:
Not sure if I'd classify this as a bug fix, but it's not exactly feature either. EPAS sometimes uses "Enhancement" or "Change" for these kinds of things in their release notes. I've used "Enhancement" before in PGD changes too.